src/components/Connect/Burner/ImportPk/index.tsx

```diff
@@ -63,10 +69,17 @@ export const ImportPk = ({ back, next }: ImportPkProps) => {
               <p>Loading...</p>
             </Loading>
           ) : (
-            <InputsWrapper>
+            <Lines>
+              <LabelWrapper>
+                <span>Please enter your private key below to continue</span>
+                <InputV2.Toggle
+                  label='Hex'
+                  onChange={setIsHex}
+                  checked={isHex}
+                />
+              </LabelWrapper>
```

Comment:
@Schwartz10 I think it looks a bit better with the toggle above the input field. This matches the order of how to fill this form:
- first decide whether to input base64 or hex
- then enter the private key